libs/ui/package.json

```diff
@@ -79,7 +79,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-vx": "workspace:*",
     "is-ci-cli": "^2.2.0",
-    "jest": "^26.6.3",
+    "jest": "26",
```

Comment:
is this an upgrade? From 26.6.3 to 26?

Reply:
I did `pnpm install -D jest@26`, which told it to just get the latest that's still 26.x.x. So the lockfile is higher than it was.